Justfile

```
@@ -16,20 +16,32 @@ build-base:
 
 build-flatpak $manifest=manifest $branch=branch:
     #!/usr/bin/env bash
-    set -xeuo pipefail
-    flatpak --user remote-add --if-not-exists flathub https://dl.flathub.org/repo/flathub.flatpakrepo
-
-    FLATPAK_BUILDER_DIR=$(realpath ".flatpak-builder")
-    BUILDER_ARGS+=(--default-branch="${branch}")
-    BUILDER_ARGS+=(--state-dir="${FLATPAK_BUILDER_DIR}/flatpak-builder")
+    set -xeo pipefail
+    FLATPAK_BUILDER_DIR="$(realpath ".flatpak-builder")"
+    BUILDER_ARGS+=("--default-branch=${branch}")
+    BUILDER_ARGS+=("--state-dir=${FLATPAK_BUILDER_DIR}/flatpak-builder")
     BUILDER_ARGS+=("--user")
     BUILDER_ARGS+=("--ccache")
     BUILDER_ARGS+=("--force-clean")
     BUILDER_ARGS+=("--install")
-    BUILDER_ARGS+=("--disable-rofiles-fuse")
     BUILDER_ARGS+=("${FLATPAK_BUILDER_DIR}/build-dir")
     BUILDER_ARGS+=("${manifest}")
 
+    if [ "${CI}" == 1 ] ; then
+        set -u
+        flatpak remote-add --if-not-exists flathub --user "https://dl.flathub.org/repo/flathub.flatpakrepo"
+        flatpak remote-add --if-not-exists gnome-nightly --user "https://nightly.gnome.org/gnome-nightly.flatpakrepo"
+        flatpak install --user -y "org.gnome.Sdk/$(arch)/master"
+        flatpak install --user -y "org.gnome.Platform/$(arch)/master"
+        flatpak install --user -y "runtime/org.freedesktop.Sdk.Extension.rust-stable/$(arch)/24.08"
+        flatpak install --user -y "runtime/org.freedesktop.Sdk.Extension.llvm20/$(arch)/24.08"
```

```suggestion
        flatpak install --user -y "org.gnome.Sdk//master"
        flatpak install --user -y "org.gnome.Platform//master"
        flatpak install --user -y "runtime/org.freedesktop.Sdk.Extension.rust-stable//24.08"
        flatpak install --user -y "runtime/org.freedesktop.Sdk.Extension.llvm20//24.08"
```
this should work, flatpak automatically figures out what march it needs to pull
https://docs.flatpak.org/en/latest/available-runtimes.html#check-software-available-in-runtimes